### DISCUSSION TITLE
20k Github Stars!!!

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/quasar#11568] [Announcements]
We made it! 20k Github Stars! 

We couldn't have done it without you, our great community. 

Thank you for getting us to this wonderful milestone. 

And, let's go for more!  😍

![image](https://user-images.githubusercontent.com/3314957/144892865-4e7d2c8d-3376-4aa7-b53f-05c8977fe1cf.png)

**Comment by hawkeye64** [comment]
Congrats to everyone that made this happen!
We have the best community coupled with the best ecosystem.
Time to party! :partying_face:

**Comment by pikil** [comment]
That's an amazing framework and you guys deserve it! Many thanks for all the effort!
